dynamodb_helpers.js

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
All of this logic is unchanged, I just moved it out of the main file. That did require adding a parameter for the database client in some cases.